app/src/main/java/io/gnosis/safe/ui/settings/owner/list/OwnerSelectionViewModel.kt

```diff
@@ -61,6 +63,7 @@ class OwnerSelectionViewModel
             val key = derivator.keyForIndex(ownerIndex)
             val addresses = derivator.addressesForPage(ownerIndex, 1)
             ownerCredentialsVault.storeCredentials(OwnerCredentials(address = addresses[0], key = key))
+            tracker.setNumKeysImported(1)
```

Comment:
Should I add a test that verifies that `tracker.setNumKeysImported(1)` was called?

Reply:
If there is a test in place, adding a `verify` method call is sufficient IMO.